Source/WebCore/rendering/style/BasicShapes.h

```diff
@@ -81,6 +82,18 @@ class BasicShape : public RefCounted<BasicShape> {
 
     virtual Type type() const = 0;
 
+    bool isCircleTypeShape() const
```

Comment:
isEllipseTypeShape? technically circles are a type of ellipse, not the other way round